docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -66,26 +66,27 @@ The tables in each of the tabs below document configuration properties that are
     |Protocol | http  | The protocol to use when building a URI to local the service endpoint|
     |Host | localhost  | The host name or IP address where the service is hosted |
     |Port | 598xx | The port exposed by the target service|
-
+    
     !!! edgey "Edgex 2.0"
         For EdgeX 2.0 the map keys have changed to be the service's service-key, i.e. `Metadata` changed to `core-metadata`
 
 === "SecretStore"
 
     |Property|Default Value|Description|
     |---|---|---|
-    |||these config values are used when security is enabled and Vault access is required for obtaining secrets, such as database credentials|
-    |Host | localhost  | The host name or IP address associated with Vault|
-    |Port | 8200  | The configured port on which Vault is listening|
+    |||these config values are used when security is enabled and `SecretStore` service access is required for obtaining secrets, such as database credentials|
+    |Type | vault  | The type of the `SecretStore` service to use. Currenly only `vault` is supported.|
+    |Host | localhost  | The host name or IP address associated with the `SecretStore` service|
+    |Port | 8200  | The configured port on which the `SecretStore` service is listening|
     |Path | /`<service-key>` | The service-specific path where the secrets are kept. This path will differ according to the given service. |
```

Comment:
Current example for core services is `<service-key>/` instead of `/<service-key>` as they are using relative service specific path now.

Reply:
Done